shuup_tests/browser/admin/test_order_list.py

```diff
@@ -39,7 +39,7 @@ def _visit_orders_list_view(browser, live_server):
     url = reverse("shuup_admin:order.list")
     browser.visit("%s%s" % (live_server, url))
     assert browser.is_text_present("Orders")
-    time.sleep(0.5)  # Wait mithril for a sec
+    wait_until_appeared(browser, css_class=".picotable-item-info")
```

Comment:
I think the second wait is ok since the picotable is actually present already and the content don't currently have anything we can attach on with `wait_until_appeared`